wildfly/web-cache-offload-infinispan/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="https://jakarta.ee/xml/ns/jakartaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
+         version="6.0">
+    <distributable></distributable>
```

Comment:
also `<distributable/>` can be used here

Reply:
Sure but re-running expensive CI checks for this to change seems odd to me. WDYT?